engineapi/backups.go

```diff
@@ -20,9 +20,9 @@ import (
 )
 
 const (
-	backupStateInProgress = "in_progress"
-	backupStateComplete   = "complete"
-	backupStateError      = "error"
+	BackupStateInProgress = "in_progress"
+	BackupStateComplete   = "complete"
+	BackupStateError      = "error"
```

Comment:
What is the reason to export these?

Reply:
this constants will be used to check the replicas restoring status in engine

https://github.com/longhorn/longhorn-manager/blob/ab225bf88ccf00fb4bb7aff3e8e24245af864dcd/manager/volume.go#L535-L542

Reply:
I see. Doesn't `volume.Status.LastBackup != engine.Status.LastRestoredBackup || engine.Spec.RequestedBackupRestore != engine.Status.LastRestoredBackup` indicate the restore is complete?

Reply:
Sorry, I think I misunderstood the update logic of the originally used field during the development process; so I will remove the judgment of isRestoring and restore the original judgment method.